utils/extensionRequests.js

```diff
@@ -25,7 +24,39 @@ const formatExtensionRequest = async (extensionRequest) => {
   return { ...body, id, timestamp, assignee };
 };
 
+const transformQuery = (dev = false, size) => {
```

Comment:
When possible, keep the default params last in the order of params.
Currently, someone can never call `transformQuery` by just passing the `size` argument.
But if you interchange the positions of the params then you can.

Reply:
Made the necessary changes.